proposals/NNNN-noncopyable-generics.md

```diff
@@ -452,6 +375,22 @@ protocol LIFOJobQueue<Event>: JobQueue {
 
 This section spells out additional details about the proposed extensions.
 
+### Conditionally-copyable types
+
+An aggregate type can only be copyable if all its stored fields are copyable.
```

Comment:
Sticking with "value" instead of "aggregate" type to contrast from classes. Adding the deinit rule as well.
```suggestion
A value type can only be copyable if all its stored fields are copyable
and it does not contain a deinitializer.
```